sample/src/main/java/com/patrykandpatryk/vico/sample/ui/ChartPager.kt

```diff
@@ -93,8 +97,11 @@ internal fun ChartPager(
             ) {
                 SwipeHint(
                     currentPage = state.currentValue + 1,
-                    pageCount = itemCount,
-                )
+                    pageCount = itemCount
```

Comment:
It seems that a trailing comma is missing here.

```suggestion
                    pageCount = itemCount,
```

Reply:
Oh, this isn't required by Kotlin. Besides, all parameters are already provided.

Reply:
While trailing commas are optional, the Kotlin style guide recommends that they be used—please see [here](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas).

Reply:
Fair enough 👍 You may want to add a code style to the project.

Reply:
It looks like `ALLOW_TRAILING_COMMA` is set to `true` in the project’s code style (see [here](https://github.com/patrykandpatryk/vico/blob/development/.idea/codeStyles/Project.xml#L15)). I’ve just added two rules connected to trailing commas to the `.editorconfig` file (see [here](https://github.com/patrykandpatryk/vico/blob/development/.editorconfig#L11-L13)).